map_server/src/image_loader.cpp

```diff
@@ -43,7 +43,7 @@
 #include <SDL/SDL_image.h>
 
 #include "map_server/image_loader.h"
-#include <tf/tf.h>
+#include <tf2/LinearMath/Quaternion.h>
```

Comment:
@tfoote what's the right thing to do here? Use `tf_{bullet, kdl, etc...}` or use bullet or kdl or eigen directly? I'd like to propose a pull request to upstream that converts this package to use `tf2`, but I don't know what to do for ROS 1.